test/Makefile

```diff
@@ -285,6 +313,7 @@ agbot-docker-prereqs: get-anax
 	mkdir -p $(AGBOT_TEMPFS)/usr/local/bin $(AGBOT_TEMPFS)/root/.colonus
 	cp $(ANAX_SOURCE)/anax $(ANAX_SOURCE)/cli/hzn $(AGBOT_TEMPFS)/usr/local/bin
 	cp -r $(CURDIR)/docker/fs/etc $(AGBOT_TEMPFS)
+	rm -r $(AGBOT_TEMPFS)/etc/vault
```

Comment:
Is this supposed to be a copy (cp)?

Reply:
No, I did this since the vault config file is not of any use in the `e2edev-agbot`. There is a similar issue with the exchange config file which gets copied to `/etc/exchange/` when the entire `/etc` directory is copied in the previous line which I think is not used anywhere.